README.md

```diff
@@ -1089,22 +1089,34 @@ Other Style Guides
     export default es6;
     ```
 
-  <a name="modules--no-wildcard"></a><a name="10.2"></a>
-  - [10.2](#modules--no-wildcard) Do not use wildcard imports.
-
-    > Why? This makes sure you have a single default export,
-    > and forces you to define the API (keeping things organized)
+  <a name="modules--define-api"></a><a name="10.2"></a>
+  - [10.2](#modules--define-api) Always define your module's API at the bottom of the file.
```

Comment:
how do you feel about saying always if you have more than 1? Having this when you only have 1 will also fource you to override the eslint as you shouldn't have a export { somthing } when you only have method